tarpc/Cargo.toml

```diff
@@ -49,7 +49,7 @@ pin-project = "1.0"
 rand = "0.8"
 serde = { optional = true, version = "1.0", features = ["derive"] }
 static_assertions = "1.1.0"
-tarpc-plugins = { path = "../plugins", version = "0.13" }
+tarpc-plugins = { path = "../plugins", version = "0.13", registry = "umbra"}
```

Comment:
Is the `registry = "umbra"` bit supposed to be here?

Reply:
Nope, coworker pushed to the wrong branch.